app/src/main/java/ch/epfl/skysync/models/calendar/CalendarModel.kt

```diff
@@ -43,7 +44,7 @@ abstract class CalendarModel<T : CalendarViewable> {
    * @param timeSlot the timeSlot coordinate of the slot
    * @param produceNewValue computes a new value as function of the coordinates and the old value
    */
-  protected fun setByDate(
```

Comment:
why did you change the access writes?

Reply:
I think I needed to use one of the methods at the some point and I didn't see why it wasn't public, I would need check exactly where.

Reply:
The inheriting classes have their own setters and getters for maintaining the underlying list. So, no need to access the methods from the abstract class directly. I think protected is fine

Reply:
I've looked and I don't use it in actually, so I'll revert it back